### PR TITLE
Argo CD concept topics (Set 2)

### PR DESCRIPTION
**Pull request by MelwynPaul**

### CODE REVIEW

File: content/en/docs/samples/argo-cd.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+***WRITER COMMENT***: I think we can skip step 5. Let the users use the `default` namespace. Thoughts?
+<br>Info: Dev content as is step 5.
+6. Under the **Source** section, do the following:
+    - **Repository URL**: https://github.com/verrazzano/verrazzano/
+    - **Revision**: `master`
+    - **Path**: Path in the repository where the Kubernetes resource definitions are listed. For example: `examples/helidon-config`
+7. Under the **Destination** section, do the following:
+    - **Cluster URL**: Select the cluster to which you want to deploy the applications.
+    - **Namespace**: Specify the namespace in which you want to deploy the applications.
+8. Click **Next**.
```
[the remark this answers is not in the document]
Comment (MelwynPaul) [reply content/en/docs/samples/argo-cd.md]: Fixed.

File: content/en/docs/samples/argo-cd.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+9. Click **Create**.
+10. If you selected the `Manual` as the **Sync Policy** option, then click **Sync**.
+<br> This creates the Argo CD application and a pictorial representation of the deployed applications is displayed.
+</br>
+
+## Verify the deployed application
+
+***WRITER COMMENT***: This section is an exact copy from the Hello Config World Helidon topic - https://verrazzano.io/latest/docs/samples/helidon-config/
+
+The Hello World Helidon microservices application implements a REST API endpoint, `/greet`, which returns a message `{"message":"Hello World!"}` when invoked.
```
[the remark this answers is not in the document]
Comment (MelwynPaul) [reply content/en/docs/samples/argo-cd.md]: Fixed.

File: content/en/docs/samples/argo-cd.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+
+**NOTE**:  The following instructions assume that you are using a Kubernetes environment such as OKE. Other environments or deployments may require alternative mechanisms for retrieving addresses, ports, and such.
+
+Follow these steps to test the endpoints.
```
[the remark this answers is not in the document]
Comment (MelwynPaul) [reply content/en/docs/samples/argo-cd.md:62]: Fixed.

**Review by rosemarymarano — APPROVED** [review]

File: content/en/docs/concepts/ArgoCD.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+---
+title: "Argo CD"
+linkTitle: Argo CD
+weight: 4
+draft: false
+---
+Argo CD is an enterprise grade GitOps Kubernetes deployment tool that uses Git repositories as the source of truth. It can leverage various declarative deployment mechanisms such as Kubernetes manifests, kustomize, ksonnet and Helm. For more information, see [Argo CD documentation](https://argo-cd.readthedocs.io/en/stable/).
+
+Argo CD is implemented as a Kubernetes controller. It monitors running applications and compares the deployed state against the desired one in Git. Argo CD reports and visualizes the differences, while providing facilities to automatically or manually update the live state with the desired target state. You can define components using Kubernetes manifests, created with the `argocd` command-line tool or the UI.
```
[the remark this answers is not in the document]
Comment (MelwynPaul) [reply content/en/docs/concepts/ArgoCD.md]: Thank you, @rosemarymarano.

File: content/en/docs/samples/argo-cd.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
+---
+title: "Argo CD"
+weight: 1
+description: "Use Argo CD to deploy and undeploy applications"
+---
+
```
[the remark this answers is not in the document]
Comment (MelwynPaul) [reply content/en/docs/samples/argo-cd.md:6]: Thank you.

**Review by rosemarymarano — APPROVED** [review]
Looks good. (My mistake: this should be -> see the Argo CD documentation.)

**Pull request merged**